README.md

```diff
@@ -17,6 +17,44 @@ production-ready Django projects quickly.
 - If you have problems with Cookiecutter Django, please open [issues](https://github.com/cookiecutter/cookiecutter-django/issues/new) don't send
   emails to the maintainers.
 
+## Table of Contents
+
+- [IntendedAudience](#IntendedAudience)
+- [Features](#Features)
+- [Constraints](#Constraints)
+- [Contributing](#contributing)
+- [License](#license)
+- [Usage](#Usage)
+- [Community](#Community)
+- [Releases](#Releases)
+- [Articles](#Articles)
+
+## IntendedAudience
+
+The purpose of this section is to outline the prerequisite knowledge that potential users need to effectively create projects
+with this application. Links will be provided to supplemental resources to help users get up to speed.
```

Comment:
You are reading a readme, not a 300 pager. So the introduction is way to long and, in my opinoin, unnecessary.